#!/usr/bin/env python
PACKAGE = "robo1"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

integration_method_enum = gen.enum(
    [ gen.const("EULER", int_t, 0, "Euler integration"),
      gen.const("RK", int_t, 1, "Runge Kutta integration")
    ],"Enum of types of integration methods")

gen.add("integration_method", int_t, 1, "Integration method", 0, 0, 1, edit_method=integration_method_enum)


exit(gen.generate(PACKAGE, "pub_odom", "method"))
